devito/operator.py

```diff
@@ -365,17 +369,21 @@ def _schedule_expressions(self, clusters, ordering):
 
         return List(body=processed)
 
-    def _insert_declarations(self, dle_state, parameters):
-        """Populate the Operator's body with the required array and
-        variable declarations, to generate a legal C file."""
+    def _specialize(self, nodes, elemental_functions):
+        """Transform the Iteration/Expression tree into a backend-specific
+        representation, such as code to be executed on a GPU or through a
```

Comment:
Nitpicking: Not sure you want to mention GPUs here, since we don't support them.